cloudsql/mysql/database-sql/cmd/app/main.go

```diff
@@ -0,0 +1,36 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"log"
+	"net/http"
+	"os"
+
+	cloudsql "github.com/GoogleCloudPlatform/golang-samples/cloudsql/mysql/database-sql"
+)
+
+func main() {
+	port := os.Getenv("PORT")
+	if port == "" {
+		port = "8080"
+	}
+	log.Printf("Listening on port %s", port)
+
+	http.HandleFunc("/", cloudsql.Votes)
```

Comment:
I'm not sure I understand why we put this in a different package. Wouldn't it make sense to all be in one?

Reply:
This is necessary to make functions work. Cloud Functions wants a non-main package. Everything else wants a main package.